### PR TITLE
nixos/prometheus-mikrotik-exporter: init

### PR DESCRIPTION
**Pull request by mmilata**
Adds package and module for https://github.com/nshttpd/mikrotik-exporter

###### Motivation for this change

Some information is not exposed through SNMP:(

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "mikrotik-exporter";
+  version = "20200210-${stdenv.lib.strings.substring 0 7 src.rev}";
```
Comment (WilliButz) [inline pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix]: Thanks for the adding the exporter with a module and test :+1:

Just one minor nitpick, could you change the version format to the following combination of "unstable-" and the ISO-8601 date for the revision?
```suggestion
  version = "unstable-2020-02-10";
```

**Review by WilliButz — CHANGES_REQUESTED** [review]

File: pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix (hunk 1 above)
Comment (Ma27) [reply pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix]: The `-unstable-` infix should be part of `(p)name` because repology: #68518

Comment (mmilata) [reply pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix]: Updated, please take a look.

Comment (WilliButz) [reply pkgs/servers/monitoring/prometheus/mikrotik-exporter.nix]: @Ma27 oh thanks for noticing, I wasn't aware of that issue :+1:

**Review by WilliButz — APPROVED** [review]

**Comment by WilliButz** [comment]
@GrahamcOfBorg test prometheus-exporters.mikrotik

**Pull request merged**